### PR TITLE
Game and Author Schema

### PR DESCRIPTION
**Pull request by AdornChoga**
In this pull request, I created the schema for game and author tables in `sql/schema.sql`.

### CODE REVIEW

File: sql/schema.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    first_name VARCHAR(255) NOT NULL,
+    last_name VARCHAR(255) NOT NULL,
+    PRIMARY KEY(id)
+);
+/* Create author table end */
+
+/* Create game table start */
+CREATE TABLE game (
+    id INTEGER GENERATED ALWAYS AS IDENTITY,
+    published_date BOOLEAN,
```
Comment (joskalenda) [inline sql/schema.sql]: - [ ] So good so far, your tables are well created however, I think the published_date variable should not be a boolean, it should be receiving a Date `published_date DATE` instead, what do you think about changing that?

**Review by joskalenda — CHANGES_REQUESTED** [review]
# STATUS: Required changes ♻️ 

- [ ]  Please check the comment under the review below

File: sql/schema.sql (hunk 1 above)
Comment (AdornChoga) [reply sql/schema.sql]: Hey, @joskalenda I actually missed that, and thanks for catching it. I will fix it right away.

**Review by joskalenda — APPROVED** [review]
# STATUS 👍  APPROVED ✅

File: sql/schema.sql (hunk 1 above)
Comment (joskalenda) [reply sql/schema.sql]: 💯 well done dude

**Pull request merged**